.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/concur/developer.concur.com
+    docker:
+      - image: circleci/ruby:2.5.1
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - devcenter-deps-{{ .Branch }}-{{ checksum "Gemfile" }}
```

Comment:
Oh, this is cool! I hadn't looked into using the cache yet, but I would imagine this will dramatically speed up build times.